src/app/core/components/screen/card/card-stats/comments/comments.component.scss

```diff
@@ -0,0 +1,5 @@
+.card__stats-comments-container {
+    display: flex;
```

Comment:
We could to add mixin with these rules and reusable it into dislikes, likes and views components